### PR TITLE
Adding cucushift rehearse - AWS Proxy

### PR DESCRIPTION
**Pull request by yunjiang29**
Resolve [OCPQE-6637](https://issues.redhat.com/browse/OCPQE-6637) Base Profile - AWS IPI cluster behind proxy

### CODE REVIEW

**Comment by yunjiang29** [comment]
@jianlinliu @gpei please review, thanks.
cc @jhou1

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-installer-rehearse.yaml [hunk 1]
```diff
@@ -34,6 +34,11 @@ tests:
   steps:
     cluster_profile: aws
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: installer-rehearse-aws-ipi-proxy
+  interval: 960h
+  steps:
+    cluster_profile: aws
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-installer-rehearse.yaml:41]: You can directly change the line 36 line to validate the new workflow so we do not have to introduce a new test grid.

**Pull request merged**